src/standard_library/field_methods.md

```diff
@@ -0,0 +1,73 @@
+
+# Field
+
+After declaring a Field, you can use these common methods on it [^migrationNote]:
+
+## to_le_bits
+
+Transforms the field into an array of bits, Little Endian.
+
+``` rust
+fn to_le_bits<N>(_x : Field, _bit_size: u32) -> [u1; N]
+```
+
+example:
+
+``` rust
+fn main(field : pub Field) {
+    let t = field.to_le_bits(32);
+    constrain t == t;
+}
+```
+
+## to_le_bytes
+
+Transforms into an array of bytes, Little Endian
+
+``` rust
+fn to_le_bytes(_x : Field, byte_size: u32) -> [u8]
+```
+
+example:
+
+```rust
+fn main(field : pub Field) {
+    let t = field.to_le_bytes(4);
+    constrain t == t;
+}
+```
+
+## to_radix
+
+Decomposes into a vector over the specificed base
+
+```rust
+fn to_radix(_x : Field, _radix: u32, _result_len: u32) -> [u8]
+```
+
+example:
+
+```rust
+fn main(field : pub Field) {
+    let t = field.to_radix(256, 4);
+    constrain t == t;
```

Comment:
This can be removed in each example.

Assuming you're trying to silence the unused variable error, I don't think it is that important but another way we can do would be removing the let binding and returning the value from main instead. Or we could keep the let binding and prefix the name with an underscore.